extensions/microsoft-authentication/package.json

```diff
@@ -31,6 +31,32 @@
       {
         "label": "Microsoft",
         "id": "microsoft"
+      },
+      {
+        "label": "Azure Cloud",
+        "id": "azure-cloud"
```

Comment:
Ok I talked to the team, and we would like to call this:

`microsoft-sovereign-cloud` and `Microsoft Sovereign Cloud` and the setting adjusted accordingly.

Reply:
Done!